onnx/checker.h

```diff
@@ -4,6 +4,7 @@
 #include <unordered_map>
 #include <unordered_set>
 #include "onnx/defs/schema.h"
+#include "onnx/defs/function.h"
```

Comment:
if this is the only one line change, then it's not needed?

Reply:
Omm this one is actually needed by lotus.. I ll remove it for the PR